utils/db.utils.js

```diff
@@ -0,0 +1,17 @@
+// const db = require("../db/connection")
+
+// exports.checkReviewExists = (review_id) => {
+//     return db
+//       .query(
+//         `
+//     SELECT * FROM reviews
+//     WHERE review_id = $1
+//     `,
+//         [review_id]
+//       )
+//       .then((review) => {
+//         if (review.rows.length === 0) {
+//             return Promise.reject({ status: 404, msg: 'Review not found!' })
+//         }
+//       });
+// }
```

Comment:
Don't commit loads of commented out code.

If you want to save it for alter add the file to .gitignore